### PR TITLE
Successfully created for loops to change the world raster resolution,…

### PR DESCRIPTION
**Pull request by EmmalineSheahan**
… rasterize polygons at each resolution, and plot species richness at each resolution. Now that we know the code is functional we can run it at different resolutions if desired

### CODE REVIEW

**Comment by dmcglinn** [comment]
Very cool! Way to stick with it! With respect to the coding style make sure you pay strict attention to the code spacing so for example you want it to look like:
```r
for(i in 1:10) {
    for(j in 1:10) {
        my_awesome_code = i * j
    }
} 
``` 
you don't want it to look like
```r
for(i in 1:10) {
    for(j in 1:10) {
    my_awesome_code = i * j
    }
} 

I know this difference may seem silly and trivial but when you get into more complex coding having the indents line up is critical to quickly telling what level of nesting within a piece of code you are in the ith loop or the jth loop for example.  Almost more important than having efficient code is to have code that is easy for other humans (including your future self) to read and decipher. Here is a style guide that I generally adhere to: http://adv-r.had.co.nz/Style.html 

I'll hold off on merging this code until you get those loops lined up correctly otherwise it looks great.

**Comment by dmcglinn** [comment]
Looks great nice job!

**Comment by dmcglinn** [comment]
Remember to execute `git pull upstream master`

File: scripts/rasterize_polygons.R [hunk 1]
```diff
@@ -15,41 +16,53 @@ res(world_raster) <- 110
...
+          temp_poly = readOGR(dsn = paste("./data/polygon/", sp_files[i], 
+                                          sep=''), layer = "OGRGeoJSON")
+          # convert projection to cea
+          temp_poly = spTransform(temp_poly, CRS("+proj=cea +units=km"))
+          # add field that will provide values when rasterized
+          temp_poly@data$occur = 1
+          # rasterize
+          sp_raster[[i]] = rasterize(temp_poly, res_list[[j]], field = 'occur')
+     }
+     raster_res_list[j] = lapply(raster_res_list, function(x) sp_raster)
```
Comment (dmcglinn) [inline scripts/rasterize_polygons.R]: Sorry I just noticed this line is still in the code. I had hinted that I don't think you want this line. What this effectively should be doing is looping through each element of `raster_res_list` and returning `sp_raster` each time. I think what you actually want is just `raster_res_list[[j]] = sp_raster`

Comment (EmmalineSheahan) [inline scripts/rasterize_polygons.R]: Ok I changed it, I'm going to run it again to make sure everything works and to update my local files while I see about fixing the raster grid to block out land masses and getting environmental raster data

**Pull request merged**